services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -3987,20 +3951,6 @@ private void handleTerminalJobStatus(
 
     boolean isTargetRegionPush = !StringUtils.isEmpty(targetedRegions);
 
-    if (!isTargetRegionPush // Push is complete for a normal batch push w/o target region push
```

Comment:
Shouldn't we keep this until the feature is fully rollout? If we remove truncation early then we'll leak some topics and in case of version rollbacks we would like run into `concurrent batch push errors` since topics were leaked?